testing/it/http/src/test/java/com/adobe/cq/wcm/core/components/it/http/PageIT.java

```diff
@@ -55,4 +64,47 @@ public void testBrandSlug() throws ClientException {
         content = adminAuthor.doGet("/content/core-components/simple-page/simple-subpage.html", 200).getContent();
         GraniteAssert.assertRegExFind(content, "<title>Simple SubPage \\| Core Components</title>");
     }
+
+    @Test
+    public void testPWAProperties() throws ClientException {
+        String content = adminAuthor.doGet("/content/core-components/simple-page.html", 200).getContent();
+        adminAuthor.getUrl("/");
+
+        // Manifest
+        GraniteAssert.assertRegExFind(content, "<link rel=\"manifest\" href=\".*/content/foo/us/en/manifest.webmanifest\" crossorigin=\"use-credentials\"/>");
```

Comment:
Added support for context path, would fail in our testing pipeline otherwise as we always use a context path.